content/applications/sales/sales/send_quotations/create_quotations.rst

```diff
@@ -0,0 +1,332 @@
+one way to add products to a quotation.
+
+Or, click :guilabel:`Catalog` to reveal a separate page, showcasing every item (and every potential
+product variant) in a neatly organized display, with items organizable by :guilabel:`Product
+Category` and :guilabel:`Attributes`.
+
+.. image:: create_quotations/product-catalog.png
+   :align: center
+   :alt: A product catalog accessible via a quotation in the Odoo Sales application.
+
+From here, simply locate the desired items, click the :icon:`fa-shopping-cart` :guilabel:`Add`
+button on the product card(s), and adjust the quantity, if needed. When complete, click the
+:guilabel:`Back to Quotation` button in the upper-left corner to return to the quotation, where the
+newly-selected catalog items can be found in the :guilabel:`Order Lines` tab.
+
+If multiple items should be presented in a more organized way on the quotation, click :guilabel:`Add
+a section`, enter a name for the section, and drag-and-drop that section heading in the desired
+location amongst the items in the :guilabel:`Order Lines` tab. The section heading appears in bold.
+
+If needed, click :guilabel:`Add a note` beneath a certain product line, to add a custom note about
+that specific product. The note appears in italics.
+
+Beneath the product lines, there are buttons that can be clicked to apply any of the following:
+:guilabel:`Coupon Code`, :guilabel:`Promotions`, :guilabel:`Discount`, and/or :guilabel:`Add
+shipping`.
+
+.. seealso::
+   - :doc:`../products_prices/ewallets_giftcards`
+   - :doc:`../products_prices/loyalty_discount`
+   - :doc:`../products_prices/prices/pricing`
+
+Optional Products tab
+---------------------
+
+Open the :guilabel:`Optional Products` tab to select related products that can be presented to the
+customer, which may result in an increased sale. For example, if the customer wants to buy a car, an
+optional product that could be offered is a *Trailer Hitch*.
+
+To learn more about the :guilabel:`Optional Products` tab on a quotation, refer to the
+:doc:`Optional products <optional_products>` documentation.
+
+Other Info tab
+--------------
+
+In the :guilabel:`Other Info` tab, there are various quotation-related configurations separated into
+four different sections: :guilabel:`Sales`, :guilabel:`Delivery`, :guilabel:`Invoicing`, and
+:guilabel:`Tracking`.
+
+.. note::
+   Some fields only appear if specific settings and options have been configured.
+
+Sales section
+~~~~~~~~~~~~~
+
+In the :guilabel:`Sales` section of the :guilabel:`Other Info` tab, there are sales specific fields
+that can be configured.
+
+.. image:: create_quotations/other-info-sales.png
+   :align: center
+   :alt: The Sales section of the Other Info tab of a quotation form in Odoo Sales.
+
+- :guilabel:`Salesperson`: assign a salesperson from the drop-down menu to be associated with this
+  quotation. The user who originally created the quotation is selected in this field, by default.
```

Comment:
Is the person in this field the one who receives a commission? Might be good to note.

Reply:
Well, that's a bit tricky. So, it kinda depends on how the individual company operates. There could be a referrer who gets a commission. Or, the commission could go to the salesperson. Or, the company could divvy such a thing up between the two (if both are even involved). So, considering that it's a bit of a "case by case" basis, I am going to leave it as this generic explanation, so as to avoid any confusion.